draft-bbs-signatures.md

```diff
@@ -439,15 +438,14 @@ Procedure:
 This operation checks that a signature is valid for a given header and vector of messages against a supplied public key (PK).
 
 ```
-result = Verify(PK, signature, header, (msg_1,..., msg_L), (H_1,..., H_L))
+result = Verify(PK, signature, header, [(H_1, msg_1),..., (H_L, msg_L)])
 
 Inputs:
 
 - PK (REQUIRED), an octet string of the form outputted by the SkToPk operation.
 - signature (REQUIRED), an octet string of the form outputted by the Sign operation.
 - header (OPTIONAL), an octet string containing context and application specific information. If not supplied, it defaults to an empty string.
-- msg_1,..., msg_L (OPTIONAL), an optional vector of octet strings.
-- H_1,..., H_L (OPTIONAL), points of G1. Generators used to commit each message.
+- [(H_1, msg_1),..., (H_L, msg_L)] (OPTIONAL), a list of generators and message pairs corresponding to signed messages. Generators must be G1 points, and the messages must be non-zero scalars mod r. If not supplied, it will default to the empty array ("[]").
```

Comment:
```suggestion
- [(H_1, msg_1),..., (H_L, msg_L)] (OPTIONAL), a list of generator and message pairs corresponding to signed messages. Generators must be G1 points, and the messages must be non-zero scalars mod r. If not supplied, it will default to the empty array ("[]").
```